cfgov/v1/tests/test_links.py

```diff
@@ -22,6 +26,41 @@ def test_gov_link(self):
         self.assertIn('class="icon-link_text"', output)
 
 
+class LinkTest(TestCase):
+
+    def setUp(self):
+
+        self.page = mock.MagicMock()
+        self.request = mock.MagicMock()
+        self.request.url = 'http://localhost:8000/about-us/blog/'
+        self.page.url = 'http://localhost:8000/about-us/blog/we-want-hear-public-about-payday-loans/'
+        self.page.live = True
+        self.page.specific.shared = True
+        self.context = mock.MagicMock()
+
+        def context_dict(name):
+            return {'request': self.request}[name]
+
+        self.context.__getitem__.side_effect = context_dict
+
+    def test_get_protected_url(self):
```

Comment:
There's no dependence here; I was testing the various possibilities that get_protected_url could return, which to my mind made sense to combine into a single unit test, since the unit being tested is the function. But if people think the different return values should each have their own test, I can certainly do that.